views/event-form.hbs

```diff
@@ -0,0 +1,190 @@
+{{#if edit}}
+  <form class="add-form" action="/{{lang}}/edit-event/{{event._id}}" method="post">
+    <h1>{{localText.edit}}</h1>
+    <input type="hidden" name="_method" value="put">
+{{else}}
+  <form class="add-form" action="/{{lang}}/add-event" method="post">
+    <h1>{{localText.add}}</h1>
+    <input type="hidden" name="_method" value="post">
+{{/if}}
+
+<span>{{localText.formInfo}}</span>
+<fieldset name="en" class="langFieldset">
+  <legend>{{localText.english}}</legend>
+  <label>{{localText.eventsName}}
+    {{#if event.en.name}}
+      <input type="text" name="name_en" value="{{event.en.name}}">
+    {{else}}
+      <input type="text" name="name_en">
+    {{/if}}
+  </label>
+
+  <label>{{localText.description}}
+    {{#if event.en.description}}
+      <input type="text" name="description_en" value="{{event.en.description}}">
+    {{else}}
+      <input type="text" name="description_en">
+    {{/if}}
+  </label>
+</fieldset>
+
+<fieldset name="ar" class="langFieldset">
+  <legend>{{localText.arabic}}</legend>
+  <label>{{localText.eventsName}}
+    {{#if event.ar.name}}
+      <input type="text" name="name_ar" value="{{event.ar.name}}">
+    {{else}}
+      <input type="text" name="name_ar">
+    {{/if}}
+  </label>
+
+  <label>{{localText.description}}
+    {{#if event.ar.description}}
+      <input type="text" name="description_ar" value="{{event.ar.description}}">
+    {{else}}
+      <input type="text" name="description_ar">
+    {{/if}}
+  </label>
+</fieldset>
+
+
+<label><span style="color:red;">*</span>
```

Comment:
Sorry I tried to explain this on the phone but didn't do a very good job - I think this drop down list needs a label like all the input boxes have, something like this:

![screen shot 2017-10-09 at 12 39 33](https://user-images.githubusercontent.com/23310908/31332422-0197a7bc-acef-11e7-8369-f126f329d247.png)

But 'Place' not 'Places' and not in red

Reply:
You already have 'eventsPlace' in the localText as a key you can use I just saw

Reply:
I thought you meant that for a sec but then I thought what you meant is what I did? do you think this way would be better?? I think it looks better without, specially when you're on the add form and you haven't selected any place yet!